DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html

```diff
@@ -0,0 +1,122 @@
+<dol-header></dol-header>
+<main-top-nav-control></main-top-nav-control>
+<div ng-show="registrationSuccess">
+  <div  class="dol-feedback-success">
+    <p class="dol-feedback-message">
+      <strong>You Have Created an Employer</strong>
+    </p>
+  </div>
+  <p>Your account is now the point of contact for the employer you just registered. </p>
+  <p>If you have any quesitons contact us at 14c-help@dol.gov.</p>
+</div>
+<div ng-show="previouslyRegistered.status">
+  <div  class="dol-feedback-error">
+    <p class="dol-feedback-message">
+      <strong>We've run into a problem</strong>
+    </p>
+  </div>
+  <p><strong>This employer already has a registered account.</strong></p>
+  <p>Please contact {{previouslyRegistered.name}}, the point of contact for this employer, to request access to the account.</p>
+  <p>If you have any questions, you can reach us at 14c-help@dol.gov.</p>
+</div>
+<form ng-hide="registrationSuccess || previouslyRegistered" ng-submit="onSubmitClick()" class="dol-form-page">
+  <fieldset class="dol-form-sub-section">
+    <legend><h3>Register an Employer</h3></legend>
+    <hr />
+    Before you can begin the application, please register an employer for this account.
+    <div class="dol-form-question-block" ng-class="validate('employer.legalName') ? 'usa-input-error' : ''">
+      <div class="dol-form-question-help">
+        <label for="legalName" class="dol-form-question-text" id="legalNameLabel">Has this employer ever previously held a 14(c) certificate?</label>
+        <span class="usa-input-error-message" role="alert" ng-show="validate('employer.hasTradeName')">{{ validate('employer.hasTradeName') }}</span>
+        <ul class="usa-unstyled-list">
+          <li>
+            <input id="hasTradeName_1" type="radio" name="hasTradeName" aria-describedby="hasTradeNameHelp" ng-value="true" ng-model="formData.employer.hasTradeName">
+            <label for="hasTradeName_1">Yes</label>
+          </li>
+          <li>
+            <input id="hasTradeName_0" type="radio" name="hasTradeName" ng-value="false" ng-model="formData.employer.hasTradeName">
+            <label for="hasTradeName_0">No</label>
+          </li>
+        </ul>
+      </div>
```

Comment:
Does this section need help text?  There's a reference to it in the code (`hasTradeNameHelp`), but there's no element with help text.  @EStriegel @simplyshang

Reply:
not sure about this. They were not in the prototype to my knowledge